src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -197,6 +197,42 @@ public static void TestECDsaPublicKey_NonSignatureCert()
         }
 
         [Fact]
+        public static void TestECDsa224PublicKey()
+        {
+            using (var cert = new X509Certificate2(TestData.ECDsa224Certificate))
+            {
+                // It is an Elliptic Curve Cryptography public key.
+                Assert.Equal("1.2.840.10045.2.1", cert.PublicKey.Oid.Value);
+
+                ECDsa ecdsa;
+
+                try
+                {
+                    ecdsa = cert.GetECDsaPublicKey();
+                }
+                catch (CryptographicException)
+                {
+                    // Windows 7, Windows 8, CentOS.
+                    return;
+                }
+
+                // Other Unix
```

Comment:
Windows 10 understands the namedCurve value, but the path we take to load it into a CngKey object doesn't proceed swimmingly.  I'll open an issue on that.

(I'll admit that at the time I wrote the comment I just didn't know which way Win10 would go :smile:).